src/components/atoms/UiRange/UiRange.vue

```diff
@@ -120,37 +120,28 @@ const props = defineProps({
     default: 1,
   },
   /**
-   * Use this props to pass attrs for decrement UiButton
+   * Use this props to pass attrs for UiNumberStepper
    */
-  buttonDecrementAttrs: {
+  numberStepperAttrs: {
     type: Object as PropsAttrs,
-    default: () => ({
-    }),
+    default: () => ({}),
   },
   /**
-   * Use this props to pass attrs for increment UiButton
+   * Use this props to pass attrs for value UiHeading
    */
-  buttonIncrementAttrs: {
+  headingValueAttrs: {
     type: Object as PropsAttrs,
     default: () => ({
+      level: 1,
```

Comment:
If we want to set some static values like level or tag for `UiHeading` we will pass it by `defaultProps`. This approach allow to override this values by `headingValueAttrs`